packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts

```diff
@@ -377,12 +377,12 @@ $panel-grid-placeholder-bg: lighten(${theme.palette.blue95}, 30%);
 $panel-grid-placeholder-shadow: 0 0 4px ${theme.palette.blue95};
 
 // logs
-$logs-color-unkown: $gray-5;
+$logs-color-unknown: $gray-5;
 
 // toggle-group
 $button-toggle-group-btn-active-bg: $brand-primary;
 $button-toggle-group-btn-active-shadow: inset 0 0 4px $white;
-$button-toggle-group-btn-seperator-border: 1px solid $gray-6;
+$button-toggle-group-btn-separator-border: 1px solid $gray-6;
```

Comment:
Need to make sure we update references to this.

Reply:
This one is used in _toggle_button_group.scss

Reply:
@jsoref Is this updated in another PR?

Reply:
Yeah, this change in another PR broke the build during Friday, so it should be okay now.